airsenal/framework/bpl_interface.py

```diff
@@ -33,17 +38,34 @@ def get_result_dict(season, gameweek, dbsession):
             next_gameweek=gameweek,
         )
     ]
+    # compute the time difference for each fixture in results
+    # to the first fixture of the next gameweek
+    result_dates = np.array(
+        [pd.Timestamp(r.fixture.date).replace(tzinfo=None) for r in results]
+    )
+    end_date = pd.to_datetime(
+        [f.date for f in get_fixtures_for_gameweek(gameweek, season, dbsession)]
+    ).min()
```

Comment:
Not played with it but should this be `.max()`? I think `time_diff` is meant to be positive so wouldn't we want everything relative to the last fixture in the gameweek?

Reply:
So I put `.min()` here because don't we want the time difference between the games to the _first_ game in the gameweek? If we had `max()`, the difference would be the latest game in the gameweek (I _think_).

Reply:
Ah, we don't use the `time_diff` for making predictions, only for fitting the model, right? That's what I was missing yesterday (I was getting confused about time differences to future matches). So yes `.min()` is right in that case 👍.